### ISSUE TITLE
Data loading UI

### ISSUE THREAD

**Issue by amaloney** [holoviz/lumen#772]
#### Is your feature request related to a problem? Please describe.

The chat interface on the rhs of the page does not contain any information for the user until data has been correctly loaded. If I click any button in this section without loaded data, I get errors.

https://github.com/user-attachments/assets/09d72398-851a-48fd-8607-4d12d64f42e8

#### Describe the solution you'd like

I would propose this section be in a drawer if no data is loaded, and have the drawer visible once the user has gone through the correct pipeline for loading data.

#### Describe alternatives you've considered

Alternatives

1. change the blue button color (a user is draw to this first)
2. disable all buttons before data is loaded, or before data is loaded have all buttons output in the chat that a user needs to first upload data

#### Additional context

The environment used for this example is given below, as well as the `app.py` file and how to download the data shown in the video.

```yaml
# environment.yaml
name: lumen-ai
channels:
    - conda-forge
dependencies:
    - python >3.9,<3.13
    - pip
    - pip:
        - panel ==1.5.4
        - panel-graphic-walker[kernel]
```

Clone, install the environment, activate it, install other dependencies, download data, _etc_.

```shell
git clone https://github.com/holoviz/lumen
cd lumen
conda env create --file environment.yaml && conda activate lumen-ai
pip install --editable .[ai-llama,sql]
wget --output-file=earthquakes.csv "https://earthquake.usgs.gov/fdsnws/event/1/query?format=csv"
```

Use panel to serve the below app code.

```shell
panel serve app.py --show
```

```python
# app.py
import lumen.ai as lmai
llm = lmai.llm.Llama()
# NOTE: not-a-file.csv does not exist.
lmai.ExplorerUI(["not-a-file.csv"], llm=llm, agents=[lmai.agents.VegaLiteAgent]).servable()
```

**Comment by philippjfr** [comment]
I think the solution here is to error on launch if the data you're asking for doesn't exist.

**Comment by ahuang11** [comment]
Fixed in #782
